test/e2e/run-test.sh

```diff
@@ -1,13 +1,18 @@
 #!/bin/bash
 
-while ! curl -f -s -o /dev/null "http://backoffice:3000"
+adminConsoleHost="localhost"
+adminConsolePort="3000"
+if [[ -z "$e2eLocal" ]]; then
+  adminConsoleHost="adminconsole"
+fi
+
+while ! curl -f -s -o /dev/null "http://$adminConsoleHost:$adminConsolePort"
 do
-    echo "[$(date --rfc-3339 seconds)] - Waiting for http://backoffice:3000"
+    echo "[$(date --rfc-3339 seconds)] - Waiting for Kuzzle Admin Connsole to be up at http://$adminConsoleHost:$adminConsolePort"
```

Comment:
Nitpicking: to make the wait more bearable, maybe we could print that message once, and then print only a dot each time we check for a connection?

Reply:
Good idea!